### PR TITLE
EHPR-149 | Additional comments for non-clinical

### PR DESCRIPTION
**Pull request by arranfw**
https://freshworks.atlassian.net/browse/EHPR-149

### CODE REVIEW

File: apps/api/src/common/helper/csv/formExport.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ export enum FormExportColumnHeaders {
   currentEmployment = 'Current Employment Type',
   employmentCircumstance = 'Employment Circumstance',
 
-  additionalComments = 'Additional Comments',
+  nonClinicalJobTitle = 'Additional Comments',
```
Comment (dbayly-freshworks) [inline apps/api/src/common/helper/csv/formExport.ts:37]: The ticket is a bit confusing, however given that the title of the field is still `Additional Comments`, I feel a bit weird changing the key.

Comment (arranfw) [reply apps/api/src/common/helper/csv/formExport.ts:37]: Yeah I was a bit conflicted, we should probably change the header now that the content of the field has changed right?

Comment (dbayly-freshworks) [reply apps/api/src/common/helper/csv/formExport.ts:37]: Yeah, I'd double check with Erica on that one. It's possible that the intent of the field has changed, designs just haven't been update yet. 
I have a suspicion that they're asking for a field with two purposes though.

**Review by fwpushan — APPROVED** [review]

**Comment by fwpushan** [comment]
@arranfw Please fix conflict

**Pull request merged**